webapp/app/partials/main/jobs.html

```diff
@@ -51,17 +51,17 @@
     <div class="result-set-body" th-clone-jobs ></div>
 </div>
 
-<div ng-show="result_sets.length == 0 && !isLoadingRsBatch.appending &&
+<div ng-show="result_sets.length != 0 && !isLoadingRsBatch.appending &&
               !isLoadingJobs && locationHasSearchParam('revision')"
               class="result-set-body unknown-message-body">
   <span ng-init="revision=getSearchParamValue('revision')">
-      <span>Waiting for a push with revision <strong>{{revision}}</strong></span>
-      <a href="{{currentRepo.url}}/pushloghtml?changeset={{revision}}"
-         target="_blank"
-         title="open revision {{revision}} on {{currentRepo.url}}">(view pushlog)</a>
-    <span class="fa fa-refresh fa-spin"></span>
-    <div>If the push exists, it will appear in a few minutes once it has been processed.
-    </div>
```

Comment:
We generally try to avoid changing things unrelated to the bug in question in the same commit - ie the indentation here should be left as it was, since otherwise it confuses what was actually being changed for this bug.